### DISCUSSION TITLE
Default Log Folder

### DISCUSSION THREAD

**Discussion by mobab-th** [sepinf-inc/IPED#1578] [General]
iped generally logs to the program folder. I can change the log folder, for example to the case folder. 
But isn't it better to log directly to the case folder by default or to copy the log to the case folder after processing? 
When processing via Docker, for example, the log is lost if you forget to change the location.

**Comment by lfcnassif** [comment]
Other users already suggested this in the past. For me it is important to have all logs in program folder and I think it is also useful for examiners when they send the case to be analyzed by other investigators. So I think it is a good idea to store the log on both locations by default (program and output folders).

**Reply by lfcnassif** [reply]
See #1579, some users have reasons to don't copy to output by default. But we could provide more log output options.
